### DISCUSSION TITLE
Create a "Flip paddle" option

### DISCUSSION THREAD

**Discussion by ManOfMeans** [Vail-CW/vail_repeater_depricated#66] [Ideas]
I recently came across an Operator who would like to begin using Vail, but is not able to use his paddle due to it being wired backward when it was built.  It seems he's been able to work with the paddle for a long time even though it was was wired backward, because both his rig and another CW website have a feature to reverse the polarity on the inputs to accommodate such instances.

Could a "Flip paddle" or a "Reverse key polarity" feature be created as a radio button or check box that enables this ability?

**Comment by nealey** [comment]
Yes, this can be added.

This should probably live near the input buttons, and will require yet another fundamental restructuring of the code, to remove the notion of a "Dit" and a "Dah" key, but rather a "left" and "right" key which are mapped to something.

**Comment by nealey** [comment]
This is issue #67
